packages/manager-components/src/components/tile/tile.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OsdsText } from '@ovhcloud/ods-components/text/react';
+import { OsdsChip } from '@ovhcloud/ods-components/chip/react';
+import { OsdsTile } from '@ovhcloud/ods-components/tile/react';
+import { OsdsLink } from '@ovhcloud/ods-components/link/react';
+import { OsdsIcon } from '@ovhcloud/ods-components/icon/react';
+import {
+  OdsHTMLAnchorElementTarget,
+  OdsHTMLAnchorElementRel,
+} from '@ovhcloud/ods-common-core';
+
+import { ODS_ICON_NAME, ODS_ICON_SIZE } from '@ovhcloud/ods-components/icon';
+import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components/text';
+import { useDynamicTranslation } from '../../translation/useDynamicTranslation';
+
+export interface Badge {
+  text: string;
+  color: ODS_THEME_COLOR_INTENT;
+}
+
+export interface ImageDetails {
+  src?: string;
+  alt?: string;
+}
+
+export interface ScTileProps {
+  href: string;
+  isExternalHref?: boolean;
+  img?: ImageDetails;
+  texts: {
+    title: string;
+    description: string;
+    category: string;
+  };
+  dataTracking?: string;
+  badges?: Badge[];
+}
+
+export const ScTile = ({
+  href,
+  isExternalHref,
+  img,
+  dataTracking,
+  badges,
+  texts,
+}: ScTileProps) => {
+  const { title, description, category } = texts;
+  const { t } = useDynamicTranslation('tile');
+
+  return (
+    <a
+      className="msc-tile-wrapper"
+      target={OdsHTMLAnchorElementTarget._blank}
+      rel={OdsHTMLAnchorElementRel.noopener}
+      href={href}
+    >
```

Comment:
**question:** Why wrapping a tile around anchor tag? I dont think it will be correct with respect to accessibilty.

Reply:
Cause all the card is clickable

Reply:
Cards are not clickable. Can you show me an example where it is clickable?